packaging/fedora/snap-mgmt.sh

```diff
@@ -8,7 +8,20 @@
 set -e
 
 SNAP_MOUNT_DIR="/var/lib/snapd/snap"
-SNAP_UNIT_PREFIX="var-lib-snapd-snap"
+SNAP_UNIT_PREFIX="$(systemd-escape -p $SNAP_MOUNT_DIR)"
+
+show_help() {
+    exec cat <<'EOF'
+Usage: snap-mgmt.sh [OPTIONS]
+
+A simple script to cleanup snap installations.
+
+optional arguments:
+  --help                           Show this help message and exit
+  --snap-mount-dir=<path>          Provide a path to be used as $SNAP_MOUNT_DIR
+  --purge                          Purge all data from $SNAP_MOUNT_DIR
+EOF
+}
```

Comment:
This is _not_ a good idea.. This is *not* meant to be run by humans, only tools. That's explicitly why it's in `%_libexecdir/snapd` rather than being in `%_bindir`.